example/simple/main.go

```diff
@@ -0,0 +1,46 @@
+// Copyright 2017 The go-github AUTHORS. All rights reserved.
+//
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+//
+// The simple command demonstrates a simple functionality which
+// prompts the user for a GitHub username and lists all the public
+// organization memberships of the specified username.
+
+package main
+
+import (
+	"context"
+	"fmt"
+	"github.com/google/go-github/github"
+)
+
+// Fetch all the public organizations' membership of a user.
+//
+func FetchOrganizations(username string) ([]*github.Organization, error) {
+	ctx := context.Background()
```

Comment:
Inline `ctx` into the only call where it's needed. It's not being used in any other way, so inlined is simpler I think, and works better for this example. I.e.:

```Go
orgs, _, err := client.Organizations.List(context.Background(), username, nil)
```